### DISCUSSION TITLE
Individual Torrent Upload Speed Limit and Peer Connection Limit

### DISCUSSION THREAD

**Discussion by ItsRainingHP** [haugene/docker-transmission-openvpn#2072] [General]
Good evening.

I would like to request an individual torrent speed and peer connection limit environment variable. I could not find if this was possible or implemented already. For example, if you set a 250 MB/s individual torrent speed variable. No single torrent goes beyond 250 MB/s while still adhering to the global maximum if set. Same concept with the individual torrent peer connection limit.

This will allow for clients intended as a seedbox to seed multiple torrents at once, without one individual torrent overpowering the others. I hope that makes sense.

Thank you for reading and again great docker!

**Comment by pkishino** [comment]
Feel free to direct this towards
https://github.com/transmission/transmission
 we don’t really have anything to do with features in the official client
